src/pages/Detailed/mobile/MobilePopularMetrics.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import Icon from '@santiment-network/ui/Icon'
+import { POPULAR_METRICS } from './utils'
+import MobileMetricCard from '../../../components/MobileMetricCard/MobileMetricCard'
+import styles from './MobilePopularMetrics.module.scss'
+
+const LS_LABEL = 'TOOLTIP_MOBILE_METRICS_SWIPES'
+
+const MobilePopularMetrics = ({ metrics, onToggleMetric, ...rest }) => {
+  const [isShow, setIsShow] = useState(false)
+
+  const wasShown = localStorage.getItem(LS_LABEL)
+
+  const hideTooltip = () => {
+    localStorage.setItem(LS_LABEL, '+')
+    setIsShow(false)
+  }
+
+  useEffect(() => {
+    if (!wasShown) {
+      setIsShow(true)
+    }
```

Comment:
`localStorage.getItem(LS_LABEL)` could be called only on component mount (currently on every render)